.goreleaser.yaml

```diff
@@ -1,3 +1,4 @@
+version: 2
```

Comment:
Can we do the v2 bump on a separate PR?

Reply:
sure

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/609